Simplenote/TextField.swift

```diff
@@ -0,0 +1,101 @@
+import Foundation
+
+
+// MARK: - TextFieldDelegate: Our custom protocol.
+//         Why: Because `shouldBeginEditing`'s API requires a non null NSText instance, which we really can't acquire
+//         within `acceptsFirstResponder`.
+//
+protocol TextFieldDelegate: NSTextFieldDelegate {
+    func controlAcceptsFirstResponder(_ control: NSControl) -> Bool
+}
+
+
+// MARK: - TextField
+//
+class TextField: NSTextField {
```

Comment:
SwiftUI has a [TextField](https://developer.apple.com/documentation/swiftui/textfield) struct. I know its scoped by framework, but would it be a best practice to use a different name?